storage/pruning/pruningStorer_test.go

```diff
@@ -821,7 +968,7 @@ func TestPruningStorer_processPersistersToClose(t *testing.T) {
 
 	ps := pruning.NewEmptyPruningStorer()
 	ps.SetNumActivePersistersParameter(3)
-	ps.AddMockActivePersisters([]uint32{10, 9, 8, 7, 6})
+	ps.AddMockActivePersisters([]uint32{6, 7, 8, 9, 10}, false)
```

Comment:
Please revert this.

Reply:
No. The test's behaviour is the same, but I've just changed the `AddMockPersisters` to add the persisters in the correct order (highest epoch - first)

Reply:
done